stories/components/LoadingStory.tsx

```diff
@@ -17,7 +17,7 @@ const LoadingPropTypesChapter = {
 
 const LoadingStory = () => (
     <section className="story loading">
-        <Loading />
+        <Loading aria-label="Loading" />
```

Comment:
Could we set "Loading" as the default `aria-label` if one is not provided?

Reply:
💯